src/Symfony/Component/Validator/Constraints/File.php

```diff
@@ -18,6 +18,8 @@
  * @Annotation
  * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
  *
+ * @property int $maxSize
```

Comment:
The scope of `$maxSize` is **protected**, so IDE assumes it cannot be accessed outside class hierarchy.
However it can actually be accessed via the magic method `__get` and `__set`, so we can add `@property` tag on the class.

Please see [Group](https://github.com/symfony/symfony/blob/befad6315bbe30f206a3e70c2c0fc4695ff9fc4b/src/Symfony/Component/Validator/Constraint.php#L28) constraint, it's doing in the same way.